web/src/settings_preferences.ts

```diff
@@ -193,8 +193,6 @@ export function set_up(settings_panel: SettingsPanel): void {
     const settings_object = settings_panel.settings_object;
     const for_realm_settings = settings_panel.for_realm_settings;
 
-    $container.find(".advanced-settings-status").hide();
```

Comment:
Would appreciate an extra look here. Posting the relevant commit message for convenience:

> This was introduced in https://github.com/zulip/zulip/commit/87635b9e32259acdecf2283623c3d13fd5d93d87 and kept
getting renamed over time and did not get deleted. Other statuses'
`.hide()` got deleted over time. I'm not sure why we were using the hide
in the first place, but manual testing shows no behaviour change when
removing it.

Reply:
Yeah that looks like dead code, thanks for noticing it!